doc/ref_cert/lfn/chapters/chapter05.md

```diff
@@ -104,40 +107,80 @@ execution and result reporting along with required artifacts and environments in
 place.
 
 #### 5.3.1 Test Case Model
+As there are more number of VNF at different levels of networking such as access,
+transport and core level as well as OSI level L0-L7. Every network functions
+provides set of pre-defined features and functionalities. so its important to
+model test cases for every functionality to identify it uniquely and use it as
+part of test flow design.
+
+As part of modeling its very important to capture the following details
+
+* Test case Name
+* Test case description
+* Virtual Network function Name
+* Network function Feature/functionality name
+* Test case input parameters
+* Test case result attributes
+* Test case version
+
+while implementing the test cases, this model would act as specification and as
+it captures the input and output, it would help while designing the test flow
+which will help to execute set of test cases in pre-defined flow.
 
 #### 5.3.2 Test case management
 
-Test suite
-
-Test flow
+* **Test case** : On-board/discover, update, disable/enable, delete
+* **Test suite** : On-board/discover, update, disable/enable, delete
+* **Test flow** : design/discover, update, disable/enable, delete
 
 #### 5.3.3 Test Execution management
 
-Different run-time support (JVM, Python, Shell, Container, Cloud VM, etc)
+* **Run-time**: One of the common nature of the test environment is heterogeneous
+and multiple vendors and open communities would provide various test tool and
+environment to support execution of test cases developed under different run-times  
+(JVM, Python, Shell, Container, Cloud VM, etc)
+
+* **RPC**: In order to enable the scaling/remote execution, it should be enabled with
+required gRPC or queue support
 
-gRPC support
+When VNF test platform execute the test cases, it captures the foot-prints of
+test case execution along with results, which are made available to user and
+integrated system for consuming.
 
 #### 5.3.4 Test Result management
 
-Search & Reporting
+* **Formats** : As part of execution management, system produces the result in JSON

```

Comment:
```system produces the result in JSON``` -> Shouldn't be enforced.

Reply:
@msidana  its required because platform should be able to communicate the result in format which LFN bading registry should be able to understand and JSON is the universal way of exchaning the information over wire. and i already mentioned in following statements that it would help to make it different format required. so i hope this is ok.

Reply:
@msidana - can we close/resolve?

Reply:
@msidana - can we close/resolve?